tests/actions.go

```diff
@@ -1369,14 +1370,12 @@ func (oa *operatorActions) cloneOperatorRepo() error {
 }
 
 func (oa *operatorActions) checkoutTag(tagName string) error {
-	cmd := fmt.Sprintf(`cd %s &&
-		git stash -u &&
-		git checkout %s &&
-		mkdir -p %s &&
-		cp -rf charts/tidb-operator %s &&
-		cp -rf charts/tidb-cluster %s &&
-		cp -rf charts/tidb-backup %s`,
-		oa.cfg.OperatorRepoDir, tagName, filepath.Join(oa.cfg.ChartDir, tagName), oa.operatorChartPath(tagName), oa.tidbClusterChartPath(tagName), oa.backupChartPath(tagName))
+	cmd := fmt.Sprintf("cd %s && git stash -u && git checkout %s && "+
+		"mkdir -p %s && cp -rf charts/tidb-operator %s && "+
+		"cp -rf charts/tidb-cluster %s && cp -rf charts/tidb-backup %s",
+		oa.cfg.OperatorRepoDir, tagName,
+		filepath.Join(oa.cfg.ChartDir, tagName), oa.operatorChartPath(tagName),
+		oa.tidbClusterChartPath(tagName), oa.backupChartPath(tagName))
```

Comment:
why use “+” in sprintf？

Reply:
use `+` to join two or more string to a single string, this is normal usage.